sdk/docs/pages/typescript-sdk/connect.mdx

```diff
@@ -0,0 +1,53 @@
+# Connecting to Sui Network
+
+The Sui TypeScript SDK provides a SuiClient class that you can use to connect to a network's JSON-RPC server. Use the `SuiClient` for all JSON-RPC read-only operations. 
```

Comment:
the SuiClient is not strictly read-only, it also supports other RPC methods like sending a transaction